web/src/Web.App/Views/LocalAuthority/Index.cshtml

```diff
@@ -0,0 +1,118 @@
+﻿@using Web.App.Domain
+@using Web.App.TagHelpers
+@using Web.App.ViewModels.Components
+@model Web.App.ViewModels.LocalAuthorityViewModel
+@{
+    ViewData[ViewDataKeys.Title] = PageTitles.LocalAuthorityHome;
+}
+
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+        <h1 class="govuk-heading-l govuk-!-margin-bottom-1">@Model.Name</h1>
+        <p class="govuk-body">
+            <a href="@Url.Action("Index", "FindOrganisation")" class="govuk-link govuk-link--no-visited-state">Change local authority</a>
+        </p>
+    </div>
+</div>
+
+<hr class="govuk-section-break govuk-section-break--m">
+
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-full">
+        <h2 class="govuk-heading-m">Schools in local authority</h2>
+        <div class="govuk-accordion" data-module="govuk-accordion" id="accordion-schools-in-authority">
+            <div class="govuk-accordion__section">
+                <div class="govuk-accordion__section-header">
+                    <h3 class="govuk-accordion__section-heading">
+                        <span class="govuk-accordion__section-button" id="accordion-schools-in-authority-heading-1">
+                            Primary schools
+                        </span>
+                    </h3>
+                </div>
+                <div id="accordion-schools-in-authority-content-1" class="govuk-accordion__section-content">
+                    <ul class="govuk-list">
+                        @foreach (var school in Model.PrimarySchools)
```

Comment:
suggestion: It feels like there is a lot of repetition here. Perhaps it would be worth using a partial view?

Reply:
Updated to use a partial in the latest commit